### PR TITLE
Rename `Mac OS X` to `macOS`

### PR DESCRIPTION
**Pull request by fabienrohrer**
Fixed partially #232

### CODE REVIEW

**Comment by fabienrohrer** [comment]
👍

File: guide/the-3d-window.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-## The 3D Window
+macOS## The 3D Window
```
Comment (omichel) [inline guide/the-3d-window.md]: This one looks wrong...

Comment (fabienrohrer) [inline guide/the-3d-window.md]: Well spotted. Fixed.

**Comment by fabienrohrer** [comment]
👍

**Pull request merged**